docs/orchestrating-elastic-stack-applications/logstash.asciidoc

```diff
@@ -458,17 +463,22 @@ spec:
       terminationGracePeriodSeconds: 604800
 ----
 
-NOTE: A https://github.com/kubernetes/kubernetes/issues/94435[known issue] in Kubernetes, means that Kubernetes may not honor `terminationGracePeriodSeconds` settings greater than 600. This may mean that a queue of a terminated Pod may not be fully drained, even when `queue.drain: true` is set and a high `terminationGracePeriodSeconds` is configured.
+NOTE: A https://github.com/kubernetes/kubernetes/issues/94435[{k8s} known issue]: {k8s} may not honor `terminationGracePeriodSeconds` settings greater than 600. 
+A queue of a terminated Pod may not be fully drained, even when `queue.drain: true` is set and a high `terminationGracePeriodSeconds` is configured.
 
-NOTE: In the technical preview, there is currently no way to drain a dead letter queue automatically before Logstash shuts down. To manually drain a Dead Letter Queue, first stop sending data to it, by either disabling the DLQ feature, or disabling any pipelines that send to a DLQ, and wait for events to stop flowing through any pipelines reading from the input.
+NOTE: In this technical preview, there is currently no way to drain a dead letter queue (DLQ) automatically before {ls} shuts down. 
+To manually drain the queue, first stop sending data to it, by either disabling the DLQ feature, or disabling any pipelines that send to a DLQ. 
+Then wait for events to stop flowing through any pipelines reading from the input.
 
 
-[float]
+[id="{p}-emptydir"]
+[discrete]
 == EmptyDir
 
 If you are not concerned about data loss, you can use an `emptyDir` volume for Logstash data.
 
-CAUTION: The use of `emptyDir` in a production environment may generate permanent data loss. Do not use with persistent queues, or dead letter queues, or when using any plugin that requires persistent storage to keep track of state between restarts of Logstash.
+CAUTION: The use of `emptyDir` in a production environment may cause permanent data loss. 
+Do not use with persistent queues (PQs), dead letter queues (DLQs), or with any plugin that requires persistent storage to keep track of state between restarts of {ls].
```

Comment:
wrt "any plugin that requires persistent storage". Are there resources we can point to for this info? Would a user know which plugins require persistent storage?

Reply:
Addressed in https://github.com/elastic/cloud-on-k8s/pull/7053#discussion_r1272569236.